pkg/cmd/server/origin/master_config.go

```diff
@@ -790,6 +790,11 @@ func (c *MasterConfig) KubeClient() *kclient.Client {
 	return c.PrivilegedLoopbackKubernetesClient
 }
 
+// OpenShiftClient returns the openshift client object
+func (c *MasterConfig) OpenShiftClient() *osclient.Client {
```

Comment:
I believe the functions just exist to document how the client will be used.  The `*Client` functions essentially always return those public fields.